reverse/reverse.py

```diff
@@ -0,0 +1,3 @@
+def reverse(word):
+    """Reverse a given word"""
+    return "".join([word[letter - 1] for letter in range(len(word), 0, -1)]).lower() if len(word) > 1 else ""
```

Comment:
I would say this is too long of a line. Break out the logic into separate lines.

Also, here's a simpler way to do it:

```
reversed_word = ""

for letter in range(len(word), 0, -1):
    reversed_word += letter

return reversed_word
```

or even simpler:

```
return word[::-1]
```